index.js

```diff
@@ -20,13 +20,9 @@ function loadCommandSpecYaml(yamlString) {
  * @return {Promise}                    Promise that resolves to the passed-in config object
  */
 async function validateCommand(commandObj) {
-    try {
-        return await SCHEMA_CONFIG.validateAsync(commandObj, {
-            abortEarly: false
-        });
-    } catch (err) {
-        throw err;
-    }
+    return SCHEMA_CONFIG.validateAsync(commandObj, {
```

Comment:
having the `await` will preserve proper stack trace on errors

Reply:
I get an error: `redundant use of await on a return value` if I add it back to the same line
https://eslint.org/docs/rules/no-return-await

Need to add with extra `const`